src/screens/Library.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+
+import Card from "../components/Card";
+import Loader from "../components/Loader";
+import Button from "../components/Button"
+
+import { getData } from "../services/Services";
+
+const Library = (props) => {
+    const [cardComponent, setCardComponent] = useState(null)
+    const [user, setUser] = useState("candidate")
+
+    useEffect(() => {
+        const type = user === "company" ? "candidates" : "labs";
+        getData(type).then(data => {
+            setCardComponent(
```

Comment:
Instead of storing a whole component in the state, you could try having the Library component properly defined and use the state to decide if you should display something!

I'm thinking you can just use `data` to decide wether to display your "`cardComponent`", or the `Loader`!